src/installer.js

```diff
@@ -1,8 +1,16 @@
 const {exec} = require('@actions/exec')
 const path = require('path')
-const semver = require('semver')
 
-module.exports = {installElixir, installOTP}
+/**
```

Comment:
Move for consistency only (can move back if you prefer). I wanted, for my sanity, to have OTP > Elixir > rebar3 everywhere.